### DISCUSSION TITLE
I'm not able to render components

### DISCUSSION THREAD

**Discussion by gabrielmelodev** [radix-ui/primitives#1408] [Help]
I need help I'm using Radix in Reactjs. but when I'm using components like Dropdown Menu or Alert Dialog it doesn't render anything blank screen all content Not loading content can help solve this problem

**Comment by andy-hook** [comment]
Hi @gabrielmelodev, unfortunately we’ll need more details before we can help. Could you provide a sandbox showing your setup and how you are using these components?
